### ISSUE TITLE
Improve sheet text for low-DPI screens

### ISSUE THREAD

**Issue by arbron** [koboldpress/black-flag#599]
![low-dpi 1](https://github.com/koboldpress/black-flag/assets/19979839/2ac5bbbd-cb9f-44f3-8b42-434364624f3c)
![low-dip 2](https://github.com/koboldpress/black-flag/assets/19979839/2cee4701-73f5-495f-888b-6a66d95ec02b)
